lib/src/main/kotlin/dev/chrisbanes/snapper/LazyList.kt

```diff
@@ -62,6 +67,7 @@ public fun rememberSnapperFlingBehavior(
     endContentPadding: Dp = 0.dp,
     decayAnimationSpec: DecayAnimationSpec<Float> = rememberSplineBasedDecay(),
     springAnimationSpec: AnimationSpec<Float> = SnapperFlingBehaviorDefaults.SpringAnimationSpec,
+    snapIndex: ((SnapperLayoutInfo, targetIndex: Int) -> Int)? = null,
```

Comment:
don't you think those two params now kinda conflicts with each other? via maximumFlingDistance you can say that I don't fling to scroll more than 100 px, but then snapIndex changes the index and how we scroll to this index even if the distance is more than 100 px.
not yet sure how to better solve it though

Reply:
Yeah, it's a tricky one. They are kind of duplicative features, but also different enough that they don't model the same thing.

We could deprecate+remove `maximumFlingDuration` and rely solely on `snapIndex`. Pager would then just override `snapIndex` and return either currrentPage or currentPage + 1 depending on the direction.

Reply:
Gah. I can't deprecate parameters so will need to create new overloads. Fun.

Reply:
yeah, add a new overload with a new param and mark deprecated the one with old params